### PR TITLE
Updated match editing to use table names

### PR DESCRIPTION
**Pull request by LPNG1**

### CODE REVIEW

**Comment by LPNG1** [comment]
> Please check two things before we merge this:
> 
> 1. Why are there so many changes in your yarn.lock without any changes in your package.json?
> 2. Make sure the table property isn't saved to the DB or sent on requests

As for the first one, I updated the packages but that wasn't needed so I'll revert that
As for the second one, I don't think it does, it hasn't previously

**Comment by idanstark42** [comment]
I don't think it does either. Please make sure

File: client/app/shared/modals/model-delete/model-delete.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (2roy999) [inline client/app/shared/modals/model-delete/model-delete.component.css]: A css of component shouldn't refer to tags outside of the component

File: client/app/shared/modals/model-edit/model-edit.component.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (2roy999) [inline client/app/shared/modals/model-edit/model-edit.component.css]: A css of component shouldn't refer to tags outside of the component

**Pull request merged**